### PR TITLE
Proto files regenerated.

### PR DESCRIPTION
**Pull request by panasevychol**
Proto files regenerated.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -113,7 +113,7 @@ safety:
 
 .PHONY: mypy
 mypy:
-	mypy $(PYCOSM_SRC_DIR) $(PYCOSM_TESTS_DIR) $(PYCOSM_EXAMPLES_DIR) setup.py --exclude 'vulture_whitelist.py|cosmpy/protos'
+	mypy $(PYCOSM_SRC_DIR) $(PYCOSM_TESTS_DIR) $(PYCOSM_EXAMPLES_DIR) setup.py
```
Comment (5A11) [inline Makefile:116]: why remove exclusion of vulture and `protos`?

File: Makefile [hunk 2]
```diff
@@ -147,7 +147,7 @@ copyright-check:
 
 .PHONY: lint
 lint:
-	black $(PYCOSM_SRC_DIR) $(PYCOSM_TESTS_DIR) $(PYCOSM_EXAMPLES_DIR) setup.py
+	black $(PYCOSM_SRC_DIR) $(PYCOSM_TESTS_DIR) $(PYCOSM_EXAMPLES_DIR) setup.py --exclude $(OUTPUT_FOLDER)
```
Comment (5A11) [inline Makefile:150]: Also exclude output folder from isort

File: Makefile (hunk 1 above)
Comment (panasevychol) [reply Makefile:116]: It's now in `setup.cfg`

File: Makefile (hunk 2 above)
Comment (panasevychol) [reply Makefile:150]: Excluded in `setup.cfg`

Comment (panasevychol) [reply Makefile:150]: Black doesn't support setup.cfg, that's why left here as the command line parameter.

**Review by 5A11 — APPROVED** [review]
lgtm

**Pull request merged**